src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1920,7 +1920,6 @@ public void EnsureLastEtagIsPersisted(DocumentsOperationContext context, long do
                     ChangeVectorUtils.ThrowConflictingEtag(lowerId.ToString(), docChangeVector, newEtag, Environment.Base64Id, DocumentDatabase.ServerStore.NodeTag);
 
                 context.LastDatabaseChangeVector ??= GetDatabaseChangeVector(context);
-                context.LastDatabaseChangeVector = ChangeVector.Merge(context.LastDatabaseChangeVector, context.GetChangeVector(mergedChangeVector), context);
```

Comment:
please make a comment here with this issue
https://issues.hibernatingrhinos.com/issue/RavenDB-20732/Deleting-a-document-that-was-resharded-cause-a-change-vector-leak